jbpm-console-ng-generic/jbpm-console-ng-data-filters-client/pom.xml

```diff
@@ -132,6 +132,25 @@
       <scope>provided</scope>
     </dependency>
 
+    <!-- test -->
+
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-all</artifactId>
```

Comment:
Nit picking here, but it would be better to use `mockito-core`. `mockito-all` is uberjar which bundles all the transitive dependencies inside the same jar.